src/packer.pkr.hcl

```
@@ -1,3 +1,10 @@
+variable "admin_pass" {
+  default     = false
+  description = "Windows Administrator password."
```

Tied into the above comment:

```suggestion
  description = "The password used to connect to the instance via WinRM."
```

---

resolved with [edadc51](https://github.com/cisagov/windows-server-packer/pull/10/commits/edadc514e2076af3cfd559c8140a4550d533b2d1)